app/src/main/kotlin/org/fossify/keyboard/services/SimpleKeyboardIME.kt

```diff
@@ -112,7 +112,7 @@ class SimpleKeyboardIME : InputMethodService(), OnKeyboardActionListener, Shared
         inputTypeClass = attribute!!.inputType and TYPE_MASK_CLASS
         inputTypeClassVariation = attribute.inputType and TYPE_MASK_VARIATION
         enterKeyType = attribute.imeOptions and (IME_MASK_ACTION or IME_FLAG_NO_ENTER_ACTION)
-        keyboard = createNewKeyboard()
+        keyboard = createNewKeyboard().adjustForEmojiButton()
```

Comment:
The `adjustForEmojiButton()` helper is already called at the return site in `createNewKeyboard()`.

```suggestion
        keyboard = createNewKeyboard()
```